src/components/Settings/Speech/Speech.component.js

```diff
@@ -77,7 +81,11 @@ const Speech = ({
               secondary={getVoiceLabel(voice)}
             />
           </ListItem>
-          <ListItem divider aria-label={intl.formatMessage(messages.pitch)}>
+          <ListItem
+            disabled={voice.voiceSource === 'cloud' ? true : false}
```

Comment:
Maybe we could add a boolean property like `isCloud` to voice objects so that we can just do `disabled={voice.isCloud}`.